components/Buttons/src/MDCButton.m

```diff
@@ -274,6 +286,8 @@ - (void)commonMDCButtonInit {
 
   // Default content insets
   self.contentEdgeInsets = [self defaultContentEdgeInsets];
+  _minimumSize = CGSizeZero;
+  _maximumSize = CGSizeZero;
```

Comment:
wont this make all the buttons invisible?

Reply:
No because the sizes are only enforced if they are greater than zero.  See [`sizeThatFits:`](https://github.com/material-components/material-components-ios/pull/2254/files/ba86df99749819e91ecac5840712d1a05aefe7c2#diff-f89cec76e933099b540c38f711df8c53R403) below.